csv/parse.ts

```diff
@@ -50,35 +50,29 @@ class Parser {
   #readLine(): string | null {
     if (this.#isEOF()) return null;
 
-    if (
-      !this.#input.startsWith("\r\n", this.#cursor) ||
-      !this.#input.startsWith("\n", this.#cursor)
```

Comment:
This condition can't be `false` as the same string can't start with both `\r\n` and `\n`.